testing/get-coursier.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+# This is a script used in CI to install coursier
+set -euxo pipefail
+
+COURSIER_URL='https://github.com/coursier/coursier/releases/download/v2.0.0/cs-x86_64-pc-linux'
+COURSIER_HASH='e2e838b75bc71b16bcb77ce951ad65660c89bda7957c79a0628ec7146d35122f'
+
+check() {
+    echo "$COURSIER_HASH  $ARTIFACT" | sha256sum --check
+}
+
+ARTIFACT="$HOME/.coursier/cs"
```

Comment:
this only wrote to the homedir before because GA couldn't cache elsewhere -- it should be in `/tmp` instead